room/compiler-processing/src/main/java/androidx/room/compiler/processing/ksp/KSFileAsOriginatingElement.kt

```diff
@@ -30,7 +30,7 @@ import javax.lang.model.type.TypeMirror
  * This wrapper class helps us wrap a KSFile as an originating element and KspFiler unwraps it to
  * get the actual KSFile out of it.
  */
-internal class KSFileAsOriginatingElement(
+internal data class KSFileAsOriginatingElement(
```

Comment:
Made this a data class to support equality for testing purposes

Reply:
sgtm